src/test/java/org/opensearch/security/dlic/dlsfls/FlsExistsFieldsTest.java

```diff
@@ -32,15 +32,17 @@ protected void populateData(Client tc) {
         tc.admin()
             .indices()
             .create(
+                // FieldExistsQuery matches documents that contain a field that indexes
```

Comment:
Thank you for the help here @reta! I have been digging into Lucene trying to figure out the issue here, but have made no noticeable progress. I need to do some more reading on what `norms` means here.

Reply:
We may need to dig a bit further to figure out why the fallback to `TermQuery` does not work, but I think it should unblock you for now, thanks @cwperks !